bin/millau/runtime/src/rialto_messages.rs

```diff
@@ -146,9 +146,10 @@ impl messages::ThisChainWithMessageLanes for Millau {
 
 	fn transaction_payment(transaction: MessageLaneTransaction<Weight>) -> bp_millau::Balance {
 		// in our testnets, both per-byte fee and weight-to-fee are 1:1
-		messages::transaction_payment_without_multiplier(
+		messages::transaction_payment(
 			bp_millau::BlockWeights::get().get(DispatchClass::Normal).base_extrinsic,
 			1,
+			FixedU128::zero(),
```

Comment:
Is there a way how we can test non-zero values on rialto<>millau? What does the multiplier depend on? Is it block utilisation?

Reply:
> Is there a way how we can test non-zero values on rialto<>millau?

No

> What does the multiplier depend on? Is it block utilisation?

This is defined by `pallet_transaction_payment::FeeMultiplierUpdate`. In P+K if block has weight > 25% of its maximal weight, then this multiplier grows. If less than 25%, then it drops (to some minimal value).